app/code/community/Bolt/Boltpay/Controller/Traits/WebHookTrait.php

```diff
@@ -103,17 +103,13 @@ protected function verifyBoltSignature($payload, $signature ) {
      */
     protected function sendResponse($httpCode, $data = array())
     {
-        ob_end_clean();
         $content = is_string($data) ? $data : json_encode($data);
         $length = strlen($content);
 
         $this->getResponse()
             ->setHttpResponseCode($httpCode)
             ->setHeader('Content-Length', $length, true)
-            ->setBody($content)
-            ->sendResponse();
```

Comment:
Let's figure out why this merchant's server is sending a double response and no other merchant is having this problem.  Probably, there will be a way to cancel any further output from Magento.

Currently, the change here does not make much logical sense and while it may help in the context of a single merchant, I am hesitant to make this change across scores of merchants.